string_assignment.js

```diff
@@ -0,0 +1,194 @@
+//# String Exercises
```

Comment:
Overall good demonstration of string properties and methods. I like the fact that you went extra mile to solve bonus questions and also used different methods to accomplish that. Keep up the good work!